NSSNamespace/User_Authentication
# changed the route and the associated controller so that user remains …

…on product list page upon logout. removed the HomeController because we never used it.

To Test:

1. git checkout fw-login
2. login as any user and make sure  you are immediately taken to the product list page.
3. logout and make sure you remain on the product list page, or are redirected there if you are not currently on it.
